examples/auth0/pages/api/protected-api.js

```diff
@@ -0,0 +1,30 @@
+import { getSession } from '@auth0/nextjs-auth0'
+
+// Serverless function
+// Protected API, requests to '/api/protected' without a valid session cookie will fail
+
+export default async function handle(req, res) {
+  const session = getSession(req, res)
+
+  if (!session) {
```

Comment:
I would leave the handler wrapped in `withApiAuthRequired`, you can just assume that `session.user` exists otherwise the `handle` function wouldn't get called. (see https://github.com/auth0/nextjs-auth0/blob/main/src/helpers/with-api-auth-required.ts#L33)

So what you had before, without the `if (!user) ...` branch

Reply:
Seems fine